geth/node/manager.go

```diff
@@ -54,15 +55,15 @@ func NewNodeManager() *NodeManager {
 }
 
 // StartNode start Status node, fails if node is already started
-func (m *NodeManager) StartNode(config *params.NodeConfig) (<-chan struct{}, error) {
+func (m *NodeManager) StartNode(config *params.NodeConfig, opts ...common.NodeOption) (<-chan struct{}, error) {
 	m.Lock()
 	defer m.Unlock()
 
-	return m.startNode(config)
+	return m.startNode(config, opts)
 }
 
 // startNode start Status node, fails if node is already started
-func (m *NodeManager) startNode(config *params.NodeConfig) (<-chan struct{}, error) {
+func (m *NodeManager) startNode(config *params.NodeConfig, opts []common.NodeOption) (<-chan struct{}, error) {
```

Comment:
Why you use `[]common.NodeOption` not `...common.NodeOption`?

Reply:
Because it's a private method and there is no reason to use `...common.NodeOption`. Using `[]common.NodeOption` is less ambiguous.

Reply:
May be it's more clear to have one common signarute for provate and Public methods?